contracts/src/AddressesRegistry.sol

```diff
@@ -25,6 +25,7 @@ contract AddressesRegistry is Ownable, IAddressesRegistry {
     ICollateralRegistry public collateralRegistry;
     IBoldToken public boldToken;
     IERC20Metadata public gasToken;
+    address public liquidityStrategy;
```

Comment:
I suppose we don't want to bring `ILiquidityStrategy.sol` yet until the mento v3 branch is merged?